### PR TITLE
Add Dockerfile, Helm Chart, Cassandra Authentication

### PR DESCRIPTION
**Pull request by addevine**
Add support for connecting to an existing Cassandra with authentication. Add Dockerfile and Helm chart for deploying on Kubernetes.

### CODE REVIEW

**Comment by addevine** [comment]
@jordangraph Actual code is identical to what you reviewed previously. Helm chart has been modified for Keycloak a bit since then, and I scrubbed the chart of any actual values and put "replace-with" directions in their place.

File: charts/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# atlas-helm-chart
```
Comment (jordangraph) [inline charts/README.md:1]: @addevine Can you add a note at the top here that identifies that all changes in the `/charts` directory were by you and the date? Similar to the note you have to put at the top of changed files.

File: charts/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dependencies:
+- name: zookeeper
+  version: "1.2.3"
+  repository: "file://../zookeeper"
```
Comment (jordangraph) [inline charts/requirements.yaml:4]: This seems odd... also, is it even pointing to the right place in the file structure? wouldn't this look in the project root vs in charts?

File: charts/README.md (hunk 1 above)
Comment (addevine) [reply charts/README.md:3]: done

File: charts/requirements.yaml (hunk 2 above)
Comment (addevine) [reply charts/requirements.yaml:4]: It does _seem_ that way, but this definitely works. I can run helm template on this and it picks up those subcharts. Also this is copied straight from Argo Sandbox repo which works :)

Comment (addevine) [reply charts/requirements.yaml:4]: helm automagic

**Review by jordangraph — APPROVED** [review]

**Comment by jordangraph** [comment]
FYI, I'm adding branch protection on `2.1.0-d1`. Can you please add a document in Confluence about how to update Atlas when new versions are released?

**Pull request merged**